pithos/pithos.py

```diff
@@ -328,6 +419,22 @@ def init_actions(self, app):
         app.add_accelerator('<Primary>d', 'win.bookmark', None)
         action.connect('activate', self.bookmark_song)
 
+    def style_context_change_handler(self, style_context):
+        #We only care if the window or icon theme has changed.
```

Comment:
__[flake8]__
* [E265] block comment should start with '# '

<!-- Comment by [SideCI](https://sideci.com) -->